docs/README.md

```diff
@@ -10,7 +10,17 @@ Slider Layout is a flexible solution for building sliders of blocks within VTEX
 
 ![](https://user-images.githubusercontent.com/27777263/70230361-e839db00-1736-11ea-9f29-7c945c10a5f7.png)
 
-:information_source: *In order to use the Slider Layout as a substitute to the [Carousel app](https://github.com/vtex-apps/carousel), check out the [Building a Carousel through lists and Slider Layout](https://developers.vtex.com/vtex-developer-docs/docs/vtex-io-documentation-building-a-carousel-using-slider-layout) documentation.*
+>*In order to use the Slider Layout as a substitute to the [Carousel app](https://github.com/vtex-apps/carousel), check out the [Building a Carousel through lists and Slider Layout](https://developers.vtex.com/vtex-developer-docs/docs/vtex-io-documentation-building-a-carousel-using-slider-layout) documentation.*
+
+## Integration with Google Analytics Enhanced Ecommerce
+
+Slider layout also gives you the possibility to integrate your component with the [Google Analytics Enhanced Ecommerce feature](https://developers.google.com/analytics/devguides/collection/ua/gtm/enhanced-ecommerce). With this integration, you will be able to set up your promotion’s ID, name, and position on the [Site Editor](https://help.vtex.com/en/tutorial/site-editor-overview--299Dbeb9mFczUTyNQ9xPe1) to be shown on a Google Analytics dashboard. To do so, you must go to your store’s Admin, click **Store Setup > CMS > Site Editor** and open the image configurations of the internal promotion you want to track.
+
+![](https://user-images.githubusercontent.com/32786712/188009083-b1a1eca3-9f59-4186-80bf-bb49e99dc364.png)
+
+> For more information about managing your page content, access [Managing page and template content](https://help.vtex.com/en/tutorial/managing-page-and-template-content--3tMbx6HXy4Fy5r9EhboG37).
+
+With everything set up, Google Analytics is now able to track your internal promotions and generate a full report of its views, clicks, click-through rate, conversions, and revenue. 
```

Comment:
```suggestion
With everything set up, Google Analytics is now able to track your internal promotions and generate reports of its views, clicks, click-through rate, conversions, and revenue. 
```

Reply:
Sugestão mínima, só pra deixar mais generalista, pq imagino que dá pra montar inúmeros relatórios com essas informações no GA.